### DISCUSSION TITLE
Welcome to octogatosconf22 Discussions!

### DISCUSSION THREAD

**Discussion by AndreaGriffiths11** [gittogethers/octogatosconf22#5] [Announcements]
### OctogatosConf22
Date: 3-9-22

In attendance:

Thank you again for your time! Let's meet for 30 minutes to:
- Discus outreach to potential customer partners
- Internal needs:
    - accounts
    - mops
    - CFP committee
    - Community partnerships
    - Any others?

- Website design (contractor)
- Quick budget overview

**Comment by AndreaGriffiths11** [comment]
Mar 9, 2022 Notes
(see something I forgot, please add it!)

📣 Attention awesome humans! We’ve just finished our kick off sync and as promised here is a quick recap, [video](https://drive.google.com/file/d/1JZop6WjCwqCICO3MLS04XKnxXutGRl8e/view?usp=sharing), and calls to action:

The format
One day, two tracks: Spanish & Portuguese

  - One welcome keynote in English (@Mariorod super would love to have!)
  - One Spanish keynote: surprise we got Mr. Empanada himself https://twitter.com/buritica!
  - One Portuguese keynote: TBD
      - Two panels per track: TBD
Talks will be 20-30 minutes, we are not officially sharing yet but, if you know someone in your network who would make a good addition to the program, please share this VIP link for them to submit a proposal: https://github.co/octogatosconf22.

The hosts
Brasil track: @lariadeoliveira and @pedrolacerda
Spanish track: @highfi13g 🙏 and tbd

Volunwinning Roles, see something you like! It's yours, just add your name ✨
Lead mod team: interacts with audience, keeps an eye in Twitch chat, moderates, suspends users as needed. (Training will be provided)
Brasil Volunteer:
Spanish Volunteer:

Accounting: helps us keep the money things in order, you don’t have to be good with numbers, just have a desire to make this the best conference ever!
Volunteer 1:
Volunteer 2:

CFP Review
Brasil: TBD
Spanish: TBD

**Comment by AndreaGriffiths11** [comment]
April 27, 2022 Notes
(see something I forgot, please add it!)

📣 Attention awesome humans! We’ve just finished our first official progress sync and as promised here is a quick recap, [video](https://drive.google.com/file/d/1hOQL98vMfioWuFACsuD5KBubXLrB2WGg/view?usp=sharing), and calls to action:

The format
One day, two tracks: Spanish & Portuguese

  - One welcome keynote in English @mariorod ✅ confirmed
  - One Spanish keynote: Juan Pablo Burtica ✅  confirmed
  - One Portuguese keynote: Nina de Hora ✅  confirmed
Talks will be 20-30 minutes, please share out CFP in your networks: [CFP page](https://sessionize.com/octogatosconf22).

Registration is OPEN: https://www.eventbrite.com/e/octogatosconf22-tickets-288227083967

The hosts
Brasil track: @pachicodes and @pedrolacerda
Spanish track: @highfi13g and tbd

Still looking to fill some of this roles:
Volunwinning Roles, see something you like! It's yours, just add your name ✨

Lead mod team: interacts with audience, keeps an eye in Twitch chat, moderates, suspends users as needed. (Training will be provided)
Brasil Lead mod Volunteer:
Spanish Lead mod Volunteer:

Accounting: helps us keep the money things in order, you don’t have to be good with numbers, just have a desire to make this the best conference ever!
Volunteer 1: @SSDGM would you be keen on helping us with this side of things?

CFP Review
Brasil: @pedrolacerda @pachicodes @lara
Spanish: @DennyPerez18 @juanpflores  TBD

CTA
Suggestions for entertainment segments @pedrolacerda, could you connect us with the DJ again, please?
Anyone keen on taking on helping plan the creative side of this?
@alebourne do we have an agency we work with for close captions?
All: Think about your favorite online conference…what made it special? Share it here

**Reply by lara** [reply]
you mentioned the wrong person :)

**Reply by lucalves** [reply]
I believe it would be @mlarasusan for Brazil CFP Review :)

**Reply by mlarasusan** [reply]
Yes, that's me hehe

**Comment by alebourne** [comment]
@AndreaGriffiths11 
Hi  I do not know of an existing approved vendor for captions. However, I have heard good reviews from people who have used "webcaptioner". Maybe it is worth researching it.

**Reply by juanpflores** [reply]
I'm pretty sure there is a vendor on the DevRel team that we can work with

**Reply by AndreaGriffiths11** [reply]
We've budgeted for interpretation now and will caption post event

**Comment by alebourne** [comment]
Also, I have learned that from an accessibility compliance point of view, having the automated captions, like the ones we tried in Spanish and Portuguese with Powerpoint fulfills that requirement. However, as we know they are not the most accurate.

**Comment by Manuekle** [comment]
🇨🇴

**Comment by ayushigeorge** [comment]
🇮🇳
